index.js

```diff
@@ -603,6 +603,11 @@ module.exports = test
 function configure ({ timeout = DEFAULT_TIMEOUT, bail = false, solo = false } = {}) {
   const runner = getRunner()
 
+  if (runner.tests.count > 0 || runner.assertions.count > 0) {
+    throw new Error('ERR_CONFIGURE_FIRST')
+    return
```

Comment:
return not needed

Reply:
whoops, I did c&p and edited, didn't realize the return